### PR TITLE
[ME-19] : Index Provider SOR

### PR DESCRIPTION
**Pull request by dmittal-dmg**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -30,6 +30,11 @@
         <dmg-common-version>1.0.31-SNAPSHOT</dmg-common-version>
     </properties>
     <dependencies>
+        <dependency>
```
Comment (dmittal-dmg) [inline pom.xml:33]: temp until we ahve our proto repo

Comment (anuragsikakollu) [reply pom.xml:33]: we have our proto repo for matching engine: [matching engine proto github repo](https://github.com/Divisions-Maintenance-Group/dmg-matching-engine-proto)

File: src/main/java/com/dmg/matchingengine/Consumer/provider/ProviderConsumerService.java [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+import org.springframework.stereotype.Service;
+
+@Slf4j
+@Service
+@RequiredArgsConstructor
+public class ProviderConsumerService {
+
+    private static final BiFunction<String, ProviderOrg, ProviderEntity> MAP = (id, val) -> {
+        ProviderEntity providerEntity = new ProviderEntity();
+        providerEntity.setId(id);
```
Comment (anuragsikakollu) [inline src/main/java/com/dmg/matchingengine/Consumer/provider/ProviderConsumerService.java:23]: need to set other required attributes in the entity before saving

Comment (dmittal-dmg) [reply src/main/java/com/dmg/matchingengine/Consumer/provider/ProviderConsumerService.java:23]: The other attributes are derived from other topics.

File: pom.xml (hunk 1 above)
Comment (dmittal-dmg) [reply pom.xml:33]: I ll refactor in my next MR. Since you have already pushed the change.

**Review by anuragsikakollu — APPROVED** [review]

**Pull request merged**